pages/products/index.tsx

```diff
@@ -0,0 +1,187 @@
+import Layout from "../../components/Layout";
+import { Wrapper } from "@cabindao/topo";
+import { styled } from "../../stitches.config";
+import Link from "next/link";
+import { ArrowRightIcon } from "@radix-ui/react-icons";
+import Image from "next/image";
+import ButtonLink from "../../components/ButtonLink";
+
+const Container = styled("section", {
+  pt: "$12",
+});
+
+const HeroContainer = styled("div", {
+  display: "flex",
+  flexDirection: "column",
+  justifyContent: "space-between",
+  height: 200,
+  position: "relative",
+  h1: {
+    position: "absolute",
+    left: 4,
+    top: "calc(50% - 22px)",
+    fontSize: 32,
+    lineHeight: "44px",
+    color: "$forest",
+    mt: 0,
+  },
+});
+
+const Row = styled("div", {
+  display: "flex",
+  justifyContent: "space-between",
+});
+
+const HeroElement = styled("div", {
+  marginTop: 2.31,
+  width: 8,
+  height: 4.62,
+  backgroundColor: "$sprout",
+  position: "relative",
+  "&:before": {
+    content: "",
+    borderLeft: "4px solid transparent",
+    borderRight: "4px solid transparent",
+    position: "absolute",
+    top: -2.31,
+    borderBottom: "2.31px solid $sprout",
+  },
+  "&:after": {
+    content: "",
+    borderLeft: "4px solid transparent",
+    borderRight: "4px solid transparent",
+    position: "absolute",
+    bottom: -2.31,
+    borderTop: "2.31px solid $sprout",
+  },
+});
+
+const InnerHeroElement = styled(HeroElement, {
+  transform: "scale(0.7,0.7)",
+  backgroundColor: "$sand",
+  zIndex: 1,
+  marginTop: 0,
+  "&:before": {
+    borderBottomColor: "$sand",
+  },
+  "&:after": {
+    borderTopColor: "$sand",
+  },
+});
+
+const ProductsHero = () => (
+  <Container>
+    <Wrapper>
+      <HeroContainer>
+        {Array(4).fill(
+          <Row>
+            {Array(17).fill(
+              <HeroElement>
+                <InnerHeroElement />
+              </HeroElement>
+            )}
+          </Row>
+        )}
+        <h1>Products built by Cabin</h1>
+      </HeroContainer>
+    </Wrapper>
+  </Container>
+);
+
+const PRODUCTS = [
+  {
+    name: "NFT Passports",
+    description:
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud.",
```

Comment:
Does the page deploy right away to production on merge to main? if so, we should probably swap out this description

Reply:
Yeah, `main` gets deployed automatically. Are you able to make branched within the CabinDAO org? We could make a products branch to work from as we make all these initial products/passports additions, then merge it all to main when ready.

Reply:
I think I'm going to set the array to empty for now so that we could merge the rest of the product page and the topo installation so that we could start migrating other components to topo